backend/apps/billing/webhooks.py

```
@@ -1,29 +1,209 @@
+            sub.status = "active"
+            sub.payment_failed_at = None
+
+            # Fetch current period end from Stripe
+            if subscription_id and not is_mock:
+                try:
+                    stripe_sub = stripe.Subscription.retrieve(subscription_id)
+                    sub.current_period_end = timezone.make_aware(
+                        datetime.datetime.fromtimestamp(stripe_sub.current_period_end)
+                    )
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)